### PR TITLE
Docker container with all dependency build for the notebook...

### PR DESCRIPTION
**Pull request by rilango**
Introduce Dockerfile to create the container with source code and dependency.
This change also include some scripts to automate docker image publishing and
helper script to start the container.

The notebooks are modified to download the required dataset(s).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+ARG BASE_IMAGE=rapidsai/rapidsai:cuda11.0-runtime-ubuntu18.04-py3.8
+
+FROM ${BASE_IMAGE}
+RUN apt update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    git python3-setuptools python3-pip build-essential
+
+RUN /opt/conda/envs/rapids/bin/pip install \
+    scanpy wget python-igraph louvain leidenalg scanpy
```
Comment (avantikalal) [inline Dockerfile:9]: 1. Are you sure that all dependencies in `https://github.com/clara-parabricks/rapids-single-cell-examples/blob/master/conda/rapidgenomics_cuda10.2.yml` are included in the container? I see only a few of them being installed here.

2. `leidenalg` is not included in `https://github.com/clara-parabricks/rapids-single-cell-examples/blob/master/conda/rapidgenomics_cuda10.2.yml` but it is included here. Is it necessary? Why?

**Review by avantikalal — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (rilango) [reply Dockerfile:9]: All these notebooks were retested on AWS yesterday to make sure this change does not break any of the notebook.

File: README.md [hunk 2]
```diff
@@ -14,6 +15,18 @@ python -m ipykernel install --user --display-name "Python (rapidgenomics)"
 
 After installing the necessary dependencies, you can just run `jupyter lab`.
 
+### Docker container
+Container with all dependency with notebook and source code are available at https://hub.docker.com/r/claraparabricks/single-cell-examples_rapids_cuda10.2-runtime-ubuntu18.04-py3.8.
```
Comment (avantikalal) [inline README.md:19]: Change to 'A container with all dependencies, notebooks and source code is available at https://hub.docker.com/r/claraparabricks/single-cell-examples_rapids_cuda10.2-runtime-ubuntu18.04-py3.8.'

Also the link doesn't seem to work right now.

**Review by avantikalal — COMMENTED** [review]
Do `build.sh` and `launch.sh` need to be in the main directory?

File: README.md (hunk 2 above)
Comment (rilango) [reply README.md]: Updated the changed container name along with link to hub.docker.com

**Review by avantikalal — APPROVED** [review]

**Pull request merged**